extensions/ql-vscode/src/variant-analysis/variant-analysis-manager.ts

```diff
@@ -184,44 +184,53 @@ export class VariantAnalysisManager
     return this.app.commands;
   }
 
-  private async runVariantAnalysisFromCommand(uri?: Uri) {
-    return withProgress(
-      async (progress, token) =>
-        this.runVariantAnalysis(
-          uri || Window.activeTextEditor?.document.uri,
-          progress,
-          token,
-        ),
-      {
-        title: "Run Variant Analysis",
-        cancellable: true,
-      },
-    );
+  private async runVariantAnalysisFromCommandPalette() {
+    const fileUri = Window.activeTextEditor?.document.uri;
+    if (!fileUri) {
+      throw new Error("Please select a .ql file to run variant analysis on");
```

Comment:
Slightly nit-picky comment: This sounds like we're running a variant analysis on a `.ql` file, which doesn't really make sense 🙃 

Maybe something like:

```suggestion
      throw new Error("Please select a .ql file to run as a variant analysis");
```
(feel free to ignore though!)